app/assets/javascripts/project_preview.coffee

```diff
@@ -0,0 +1,14 @@
+root = exports ? this
+
+root.sendPreview = ->
+  sform = $($('[data-js="project_form"]')[0].elements).not('input:hidden[name=_method]').serialize()
+  $.ajax
+    type: 'POST'
+    url: '/projects/preview'
+    data: sform
+    success: (data) ->
+      $('#preview_pane').html data
+      console.log data
```

Comment:
Super stuff using Ajax! Nice one 👍 

I think it makes sense to remove the `console.log` call though, how about you?